src/main/scala/com/evolutiongaming/conhub/ConHubImpl.scala

```diff
@@ -122,6 +122,14 @@ object ConHubImpl extends LazyLogging {
         metrics.onTell(msg)
       }
     }
+
+    ConfigFactory.load().getOpt[String]("evolutiongaming.conhub.role") match {
```

Comment:
once again please never use `ConfigFactory.load()` directly, use the one provided.

Reply:
I thought we should check `Cluster(system).selfRoles` for existence

Reply:
This also duplicates code in `SendMsg`, what if we parse settings once and pass role as argument to `SendMsg` ?